app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,12 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @posts = Post.where(author_id: @user.id).order(created_at: :desc)
```

Comment:
- [ ] <b>REQUIRED:</b> if you want to get the posts that belong to this user, you can simply get them by:

```ruby
@post = @user.posts